lib/chef/resource.rb

```diff
@@ -1382,9 +1385,16 @@ def self.action(action, &recipe_block)
     #   created resource with its identity values filled in.
     #
     def self.load_current_value(&load_block)
+      include LoadCurrentValueDSL
```

Comment:
OK, I relented and just added the method with a note in the method that it is used inside `load_current_value`.